src/views/Lessons/Lessons.tsx

```diff
@@ -43,7 +43,7 @@ export const Lessons: FC = () => {
           title="Distance sensors"
           description="Dolores et rerum hic quos sed at. Eos magnam laboriosam est repellat non. Dolores nihil suscipit quod voluptate."
           buttonTextPrimary="View Challenges"
-          buttonDestPrimary="#"
+          buttonDestPrimary="1"
```

Comment:
I thought this should be 1, 2, 3, 4, 5.. but it was @owen26 who sets up those pages anyway so I'm not sure if it was placeholder or not? If it was real pages, then might want to use them?

Reply:
for now I think making them all `1` is fine, we should have a follow up PR unify data loading machanism, then this `buttonDestPrimary` will be dynamically set to real lesson ID fetched from data source.